components/eam/src/chemistry/modal_aero/aero_model.F90

```diff
@@ -824,7 +826,12 @@ subroutine aero_model_wetdep(dt, dlf, dlf2, cmfmc2, state,                    &
     ! for prognostic modal aerosols the transfer of mass between aitken and 
     ! accumulation modes is done in conjunction with the dry radius calculation
     call t_startf('calcsize')
-    call modal_aero_calcsize_sub(state, dt, pbuf, ptend)
+    do icnst = 16, pcnst
```

Comment:
Could the '16' above be changed into a parameter?

Reply:
Thanks! I was going to make the "uses" of `ptr2d_t` and getting cloud borne mmr consistent in the next PR but I did that in this PR to address your comment.